### DISCUSSION TITLE
Can I modify the layout for all to be visible on screen?

### DISCUSSION THREAD

**Discussion by Davoda1** [rakshasa/rtorrent#1268] [Q&A] [answered]
Sorry, but I couldn't find any information about it at all and this is my last resort to post here my question.

As I couldn't fine any references for this question I have no idea what the possibilities are. At least for me, not everything fits on the screen and so it's cut off and completely unacceptable as seen on attached image.
![Screenshot from 2024-03-18 14-09-34](https://github.com/rakshasa/rtorrent/assets/58233352/4da153f7-9b42-4eb6-968e-8185756b1bfd)

So the question is if there is a way around it or is it possible to modify?
Thank you for any help or advice.

**Comment by kannibalox** [comment, marked as answer]
It's not currently possible to modify it via the standard configuration, so if you can't make the window larger, the next best workaround is to use a smaller font.

Alternatively you could use third party tools via the XML-RPC API to get the same information.

**Reply by Davoda1** [reply]
Thank you, I don't think it would be the hardest to modify the code for myself, but I guess I'll just live with it for now.
Correct me if I'm wrong but on the details tab all information is visible, but I'm still a bit disappointed that there is no easy way to fit it.

Thank you for the help and maybe in an update there will be an easy way added to change it. :)
